keyboards/keebio/iris/keymaps/gary/keymap.c

```diff
@@ -0,0 +1,47 @@
+#include "gary.h"
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  [_QWERTY] = LAYOUT_kc(
```

Comment:
Layout macros that automatically prepend `KC_` are discouraged, as they do not play well with the Configurator, and make some of the more advanced keycodes such as `MO()` and `LT()` harder to use, since you need to manually define aliases beginning with `KC_`.

As this is a personal keymap, it doesn't matter so much, but in future, these layouts will probably be going away.

Reply:
oh, this is good to know.  Thanks for the heads up.